src/main/java/order/controller/OrderController.java

```diff
@@ -0,0 +1,85 @@
+package order.controller;
+
+import java.util.List;
+import order.domain.Order;
+import order.domain.OrderResult;
+import order.domain.Orders;
+import order.domain.ServiceMenu;
+import order.domain.dto.OrderResponse;
+import order.domain.dto.ServiceResponse;
+import order.global.util.OrderInputParser;
+import order.view.InputView;
+import order.view.OutputView;
+
+public class OrderController {
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public OrderController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void run() {
+        Orders orders = generateOrders();
+        OrderResult orderResult = generateOrderResult(orders);
+        ServiceMenu serviceMenu = generateServiceMenu(orders);
+
+        displayOrderSummary(orders, orderResult, serviceMenu);
+
+        inputView.closeScanner();
+    }
+
+    private void displayOrderSummary(Orders orders, OrderResult orderResult, ServiceMenu serviceMenu) {
+        displayOrderDetails(orders);
+        displayTotalAmount(orderResult);
+        displayDeliveryFee(orderResult);
+        displayServiceMenu(serviceMenu);
+        displayFinalAmount(orderResult);
+    }
+
+    private void displayFinalAmount(OrderResult orderResult) {
+        int finalAmount = orderResult.calculateFinalAmount();
+        outputView.printFinalAmount(finalAmount);
```

Comment:
```suggestion
outputView.printFinalAmount(orderResult.calculateFinalAmount());
```
예를들어 이와 같은 형식으로 바꾸면 더 간결해보이는 것 같습니다
orderResult.calculateFinalAmount() 메서드 명에서 finalAmount를 얻는다는 것을 그리 어렵지 않게 알 수 있을 것 같거든요

Reply:
오호! 짚어주셔서 감사합니다. 앞으로 인라인을 최대한 고려해보겠습니다. 사람마다 인식하는 차이가 있지만, 짚어주신 부분은 확실히 와닿을 수 있겠어요!